addons/web/static/src/core/orm_service.js

```diff
@@ -125,6 +125,14 @@ export class ORM {
         return this.call(model, "create", [state], { context: ctx });
     }
 
+    nameGet(model, ids, ctx) {
+        validatePrimitiveList("ids", "number", ids);
```

Comment:
now that I see this, we should use the validate function exported by owl!!!

Reply:
I wrote a note in our pad, we'll do that later